src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let results = days * 40;
```

Comment:
The variable `results` can be declared as `const` since it is not reassigned after its initial assignment.